### PR TITLE
Removes SBT dependency from core module

### PR DESCRIPTION
**Pull request by fedefernandez**
This PR removes the SBT dependency from the core module. The biggest change is that the runnable classes are moved from the core to the plugin, so it can lead some incompatibility issues.

### CODE REVIEW

File: core/src/main/scala/sbtorgpolicies/io/io.scala [hunk 1]
```diff
@@ -53,7 +53,7 @@ package object io {
 
       def toPath: Path = get(filename)
 
-      def toFile: File = file(filename.fixPath)
+      def toFile: File = new File(filename.fixPath)
```
Comment (juanpedromoreno) [inline core/src/main/scala/sbtorgpolicies/io/io.scala]: `IO.file` to be consistent?

File: core/src/main/scala/sbtorgpolicies/io/io.scala [hunk 2]
```diff
@@ -63,4 +63,47 @@ package object io {
            else File.separator)
     }
   }
+
+  object IO {
+
+    def file(path: String): File = new File(path)
```
Comment (juanpedromoreno) [inline core/src/main/scala/sbtorgpolicies/io/io.scala]: IO.file

File: core/src/main/scala/sbtorgpolicies/io/io.scala [hunk 3]
```diff
@@ -63,4 +63,47 @@ package object io {
...
+      array
+    }
+
+    def write(file: File, content: String, charset: Charset = Charset.forName("UTF-8")): Unit = {
+      val writer = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(file, false), charset))
+      writer.write(content)
+      writer.close()
+    }
+
+    def relativize(base: File, file: File): Option[String] = {
```
Comment (juanpedromoreno) [inline core/src/main/scala/sbtorgpolicies/io/io.scala]: Can we use https://github.com/47deg/sbt-org-policies/blob/e7ea1334edc0b12504c852fbac766acdf14a1028/core/src/main/scala/sbtorgpolicies/github/GitHubOps.scala#L64 somehow?

File: core/src/main/scala/sbtorgpolicies/io/io.scala (hunk 2 above)
Comment (fedefernandez) [reply core/src/main/scala/sbtorgpolicies/io/io.scala]: Inception?

**Comment by fedefernandez** [comment]
Thanks for the review. We need to avoid the use of `sbt.IO`, this is why we can't use it in those files.

**Pull request merged**